.cicd/test.sh

```diff
@@ -178,5 +178,8 @@ function check_creates_template() {
   check_string_existence "--set modules.text2vec-transformers.passageQueryServices.$modtransformers.enabled=true --set modules.text2vec-transformers.passageQueryServices.$modtransformers.readinessProbe.periodSeconds=988888888888" "periodSeconds: 988888888888"
   done
 
+  _settingScrapePromMetrics="--set env.PROMETHEUS_MONITORING_ENABLED=true --set serviceMonitor.enabled=true"
+  check_setting_has_value "$_settingScrapePromMetrics" "name: metrics" "containerPort: 2112"
+  check_setting_has_value "$_settingScrapePromMetrics" "apiVersion: monitoring.coreos.com/v1" "kind: ServiceMonitor"
   echo "Tests successful."
```

Comment:
Can you also add 3 more test:
1. `"--set env.PROMETHEUS_MONITORING_ENABLED=false --set serviceMonitor.enabled=true"`
2. `"--set env.PROMETHEUS_MONITORING_ENABLED=true --set serviceMonitor.enabled=false"`
3. `"--set env.PROMETHEUS_MONITORING_ENABLED=false --set serviceMonitor.enabled=false"`

Reply:
Sure, I can, add by [this commit](https://github.com/weaviate/weaviate-helm/pull/145/commits/d415a8bc4e7c0a55716bb77a1814a8eda42de319)